src/mpi/coll/ialltoallv/ialltoallv_tsp_scattered_algos.h

```diff
@@ -50,13 +50,12 @@ int MPIR_TSP_Ialltoallv_sched_intra_scattered(const void *sendbuf, const int sen
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIR_TSP_IALLTOALLV_SCHED_INTRA_SCATTERED);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIR_TSP_IALLTOALLV_SCHED_INTRA_SCATTERED);
 
-    if (bblock <= 0) {
+    /* Alltoall: each process need send #size msgs and recv #size msgs
```

Comment:
This was a previous (irrelevant) fix before I really understood the code. I am reverting to original code and adding comment to clarify the purpose.